.travis.yml

```diff
@@ -13,10 +13,16 @@ matrix:
       script: ./tools/checkJSCurrent.sh
       node_js:
         - "10"
-    - name: "Old Release Check"
-      r: oldrel
-    - name: "Current Release Check"
-      r: release
+    - name: "3.2"
+      r: 3.2
+    - name: "3.3"
```

Comment:
Electrons are cheap these days, but I'm wondering if we need to test all the incremental versions in between. What if we just tested on 3.2, current, and devel? Are the others necessary?